src/client.js

```diff
@@ -9,9 +10,11 @@ import { template, templateStyles } from './app/lib/joinUsTemplate';
 const data = window.SIMORGH_DATA || {};
 const root = document.getElementById('root');
 
-Loadable.preloadReady().then(() => {
-  hydrate(<ClientApp data={data} routes={routes} />, root);
-});
+if (matchRoutes(routes, window.location.pathname).length > 0) {
```

Comment:
Can we add an inline comment to explain why we're doing this, I can easily imagine someone coming along and removing it